yfmos.py

```diff
@@ -371,7 +371,7 @@ def __exec_B0(self, hostArg, b0, config, profile):
 
         buffer = BytesIO()
         c = pycurl.Curl()
-        url = '%s/cm?cmnd=Backlog%%20%s' % (host, b0.replace(' ', '%20'))
+        url = '%s/cm?cmnd=Backlog%%20%s%%3B%%20RfRaw%%200' % (host, b0.replace(' ', '%20'))
```

Comment:
Make this an option in the configuration file

Reply:
Ok. Does the rest of the URL change work for you? Assume default should be without the RfRaw 0 to preserve original behaviour?

I am also experiencing an issue using my modified code. Once the bridge has been running for a while issuing a command causes the bridge to emit a continuous beep that must be resolved by reset. After a restart the commands work fine with no beep.

Reply:
Yes, the new endpoint without `RfRaw 0` is fine.

I also have this beep/reboot problem sometimes, but haven't investigated it any further. The problem could be an invalid RfRaw command or a bug in the [firmware](https://github.com/Portisch/RF-Bridge-EFM8BB1).

Reply:
I have raised an issue on the Portisch firmware project here: https://github.com/Portisch/RF-Bridge-EFM8BB1/issues/146#issue-549824012